daemon/src/main/scala/io/gearpump/cluster/local/LocalCluster.scala

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
The shell command Local.sh is a shortcut tool to create a cluster, the cluster will only used one machine. It is still a standalone cluster. The user code is not colocated with that JVM process.

The LocalCluster.scala is a util to create a in-process cluster, it will share the same JVM as user code.